unicode/unicode/urls.py

```diff
@@ -14,8 +14,16 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from django.conf import settings
+from django.conf.urls.static import static
+from django.contrib.staticfiles.urls import staticfiles_urlpatterns
+
+
+admin.autodiscover()
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
-]
+    path('admin/', admin.site.urls,),
+    path('problems/', include("problems.urls"))
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```

Comment:
@kailynw , what exactly is going on right here? It seems like you're telling django where your static files are or something? Given how crucial static files are to websites there must be a neater way to do this.
Got any docs you can link to?